apps/site/lib/site/trip_plan/itinerary_row_list.ex

```diff
@@ -49,9 +49,9 @@ defmodule Site.TripPlan.ItineraryRowList do
         ]
   defp get_rows(itinerary, deps, opts, alerts) do
     rows =
-      for leg <- itinerary do
+      for {leg, index} <- Enum.with_index(itinerary.legs) do
         leg
-        |> ItineraryRow.from_leg(deps)
+        |> ItineraryRow.from_leg(deps, Enum.at(itinerary.legs, index + 1))
```

Comment:
Is there always going to be a `index + 1` element? or will we run into 'out of bounds' scenarios at some point?

Reply:
No and yes! `Enum.at` for an out of bounds index should return `nil` in that case.